mantidproject/mantid
# OFFSPEC IDF deprecated in favour of new IDF

The current OFFSPEC IDF was incorrect and therefore will be replaced with a new definition. The current IDF will now be deprecated and named `OLD_OFFSPEC_Definition.xml`. The new definition should be selected by Mantid in favour of `OLD_OFFSPEC_Definition.xml`.

**To Test:**
- Firstly make sure that System tests have passed as there may be some references to component positions that have since changed.
- Create a workspace with `CreateWorkspace` algorithm in mantid
- Run `LoadInstrument` with the workspace you just created as the `OutputWorkspace` and "OFFSPEC" as the `InstrumentName`
- Provided the deprecation has been successful the valid from date should be `2012-01-02 00:00:00` and this can be seen from the logging window in mantid (you might have to set log level to Debug)
- If the deprecation has not worked you will see the valid from date will be `2012-07-10 22:00:00`

Fixes #14281
